### PR TITLE
Hero bar and updates page content updates

### PR DESCRIPTION
**Pull request by MEspositoE14s**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-3483

## 🛠 Changes

Content updates to the hero bar and updates page (See [here](https://confluence.cms.gov/pages/viewpage.action?spaceKey=DAPC&title=%28V2%29+DPC+Onboarding+Pause+Update) for requested changes).

## ℹ️ Context for reviewers

Asked to make updates detailing our current onboarding pause for end users.

## ✅ Acceptance Validation

Tests pass and content matches what was requested.  

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: common/update-page.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ button: DPC Google Group
...
+The Data at the Point of Care (DPC) API supplements patients' health records with data from Fee-For-Service (FFS) Medicare claims. Thanks to collaboration and feedback from our initial pilot group, we're implementing an improved onboarding experience. During this process, **we've paused onboarding and new applications to DPC**. We'll provide an **update** in **July of 2023** on the length of the pause and changes to the onboarding process.
 
 Here's how the temporary pause may affect you:
 
-**If you're interested in DPC, but haven't requested access**
+**If you're interested in DPC, but haven't submitted an application**
 
-Please join us in the [Data at the Point of Care community](https://groups.google.com/d/forum/dpc-api). We'll provide regular updates there, including the date when onboarding will resume.
+We are not taking new applications for production data during the pause. However, you can still request access to the sandbox. We also encourage you to join the [Data at the Point of Care community](https://groups.google.com/d/forum/dpc-api). We'll provide regular updates there, including the date when onboarding will resume.
```
Comment (kyeah) [inline common/update-page.md:18]: There's a link in the confluence page for "request access to the sandbox" — it goes here: https://sandbox.dpc.cms.gov/users/sign_up

**Review by kyeah — COMMENTED** [review]
One discrepancy I believe, otherwise looks good to me 👍

**Comment by kyeah** [comment]
Oh wait, I saw in the comments Anna linked to a V2.1 content changes document that supercedes the V2 document, is that the one we need to be using? 

https://confluence.cms.gov/pages/viewpage.action?spaceKey=DAPC&title=%28V2.1%29+DPC+Onboarding+Pause+Update+-+new+applications

**Comment by MEspositoE14s** [comment]
> Oh wait, I saw in the comments Anna linked to a V2.1 content changes document that supercedes the V2 document, is that the one we need to be using?
> 
> https://confluence.cms.gov/pages/viewpage.action?spaceKey=DAPC&title=%28V2.1%29+DPC+Onboarding+Pause+Update+-+new+applications

Good catch, that was news to me!  The Jira ticket only has the link to V2 so that's what I used.  I'll make the changes and get a new commit up shortly.

**Review by kyeah — APPROVED** [review]
Looks good, thank you! 🌟

**Pull request merged**